contributors/chairs-and-techleads/teachnical-lead.md

```diff
@@ -0,0 +1,75 @@
+# Technical Lead
+
+## Role Description
+
+### About
+
+Target of this document is to define and outline the Technical Lead role within
+the Kubernetes community. The document can be used as guidance for Special
+Interest Groups (SIGs) to onboard new Technical Leads as well as clarifying the
+expectations associated with this role.
+
+### Abstract
+
+The Technical Lead role in Kubernetes is an optional role that each SIG can
+choose to implement as part of its [governance
+model](https://github.com/kubernetes/community/blob/3b283852567872e38f0b97beb20d918667882754/committee-steering/governance/sig-governance.md#roles).
```

Comment:
suggest using one of these URLs that will link to the latest content:

* absolute via redirect: http://git.k8s.io/community/committee-steering/governance/sig-governance.md#roles 
* relative: /committee-steering/governance/sig-governance.md#roles

Reply:
Fixed by using the absolute URL :+1: